### ISSUE TITLE
Get rid of columns?

### ISSUE THREAD

**Issue by tripu** [w3c/tr-pages#27]
And instead, organise fields of specs in a tabular way, in single rows.

/cc @bert-github.

**Comment by tripu** [comment]
This might work well on wide screens (when there's enough width for: title, type, tags, status&nbsp;&mdash;&nbsp;at least).

On smaller screens, we'd have to either truncate information heavily (bad idea) or change the `display` properties of certain elements so that they wrap and take their own new line, etc.

**Comment by wseltzer** [comment]
columns are non-intuitive to me

**Comment by chaals** [comment]
@tripu said

> On smaller screens, we'd have to either truncate information heavily (bad idea) or change the display properties of certain elements so that they wrap and take their own new line, etc.

Yeah, wrapping some is better than truncating... until you reach "TMI" and have to truncate some information.

**Comment by dret** [comment]
agreed that tables at some point become tricky. on the other hand, just displaying a bunch of blocks for everybody because on some displays tables are tricky is an odd way to approach design. for most people looking for constantly updated feeds of information, some form of list is by far the most useful, in particular if the info is uniform and i can quickly go through the list and compare dates, for example, and see which dates are the new ones. in the new block-based display, this is much harder to do.

**Comment by dret** [comment]
for context: until this week, https://www.w3.org/TR/tr-date-all used to be a simple and well-designed table view of recent publications, very easy to use and read. that has disappeared, and the URI now redirects to https://www.w3.org/TR/, which only shows a bunch of blocks that are much harder to read.
